deployments/ansible/roles/collector/README.md

```diff
@@ -56,27 +74,35 @@ how to use the role in a playbook with minimal required configuration:
 - `splunk_otel_collector_config`: Splunk OTel Collector config YAML file. Can be set to 
   `/etc/otel/collector/gateway_config.yaml` to install the collector in gateway
   mode. (**default:** `/etc/otel/collector/agent_config.yaml`)
+  For Windows this is the destination path of the collector config file on the node. 
+  The SPLUNK_CONFIG environment variable will be set with this value for the collector service.
+  (**default:** `%ProgramData%\Splunk\OpenTelemetry Collector\agent_config.yaml`)
 
 - `splunk_otel_collector_config_source`: Source path to a Splunk OTel Collector config YAML 
   file on your control host that will be uploaded and set in place of
   `splunk_otel_collector_config` in remote hosts. Can be used to submit a custom collector 
   config, e.g. `./custom_collector_config.yaml`. (**default:** `""` meaning 
   that nothing will be copied and existing `splunk_otel_collector_config` will be used)
-
+  For Windows this file will be copied to the $splunk_otel_collector_config path on the node.
+  (**default:** `%ProgramFiles%\Splunk\OpenTelemetry Collector\agent_config.yaml`)
```

Comment:
Same Q as above, another definition for Windows is confusing. The parameter should have the same meaning and behaviour across platforms